.github/workflows/Lib edFi.security.dataaccess.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       shell: pwsh
     - name: Run Unit tests
       run: |
-        .\build.githubactions.ps1 test -Configuration ${{ env.CONFIGURATION }} -Solution "Application/EdFi.Security.DataAccess/EdFi.Security.DataAccess.sln" -ProjectFile "Application/EdFi.Security.DataAccess/EdFi.Security.DataAccess.csproj"
+        .\build.githubactions.ps1 test -Configuration ${{ env.CONFIGURATION }} -Solution "Application/EdFi.Security.DataAccess/EdFi.Security.DataAccess.sln" -ProjectFile "Application/EdFi.Security.DataAccess/EdFi.Security.DataAccess.csproj" -TestFilter "DataAccessIntegrationTests"
```

Comment:
This is a little redundant since the integration tests will result in "inconclusive" if the database engine is not configured in appsettings.json. However, it felt safer to specifically exclude them from the DataAccess library build actions since none of the tests are expected to run during either.